cmake/Packaging.cmake

```diff
@@ -139,6 +138,48 @@ ly_install(FILES ${_cmake_package_dest}
     COMPONENT ${CMAKE_INSTALL_DEFAULT_COMPONENT_NAME}
 )
 
+# Do inital check on workspace path in the event that LY_3RDPARTY_PATH is misconfigured
+file(REAL_PATH "${CMAKE_CURRENT_SOURCE_DIR}/.." _root_path)
+if (EXISTS "${_root_path}/3rdParty/packages")
+    set(CPACK_LY_3P_PACKAGE_DIRECTORY "${_root_path}/3rdParty/packages")
+else()
+    set(CPACK_LY_3P_PACKAGE_DIRECTORY ${LY_3RDPARTY_PATH}/packages)
+endif()
+
+# Scan the engine and 3rd Party folders for licenses
+file(TO_NATIVE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/python/python.cmd" _python_cmd)
+file(TO_NATIVE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/scripts/license_scanner" _license_script_path)
+
+set(_license_script "${_license_script_path}/license_scanner.py")
+set(_license_config "${_license_script_path}/scanner_config.json")
+
+set(_license_scan_path "${CMAKE_CURRENT_SOURCE_DIR}" "${CPACK_LY_3P_PACKAGE_DIRECTORY}")
```

Comment:
this `Packaging.cmake` is executed as part of configuration, not as part of packaging
So there is no need to name this variable `CPACK_`

Reply:
I'll rename and move this to PreBuild